src/Helper/ArrayHelper.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Helper;
+
+
+/**
+ * Class ArrayHelper.
+ */
+class ArrayHelper
+{
+    private function __construct()
```

Comment:
The constructor is un-necessary.

Reply:
Being `private`, it prevents class instanciation

Reply:
Oh nice one!

Reply:
😘